src/import/import_excel_data.py

```diff
@@ -40,15 +74,15 @@ def read_replace_excel_table(database, metadata, filename, tablename, sheet_numb
         table = metadata.tables[tablename]
         table.drop(database)
 
-    df1.to_sql(tablename, database)
+    df1.to_sql(tablename, database, dtype=dtype)
     total_seconds = time.time() - start
     log.info("Imported {} {} items in {:.2f} seconds".format(len(df1.index), tablename, total_seconds))
 
 
-planex_file = '{}dump-panos-4-12-2017.xlsx'.format(DATA_DIR)
+planex_file = '{}dump-planex-20180620.xlsx'.format(DATA_DIR)
```

Comment:
This value seems to be repeated in download_objectstore. Could this value be moved to a config file or whatsoever?